eng/pipelines/templates/jobs/sdk-build.yml

```diff
@@ -58,7 +58,7 @@ jobs:
     templateContext:
       sdl:
         binskim:
-          analyzeTargetGlob: +:f|eng\**\*.props;+:f|artifacts\bin\**\*.dll;+:f|artifacts\bin\**\*.exe;-:f|artifacts\bin\**\msdia140.dll;-:f|artifacts\bin\**\pgort140.dll;-:f|artifacts\bin\*Tests\**;
+          analyzeTargetGlob: +:f|eng\**\*.props;+:f|artifacts\bin\**\*.dll;+:f|artifacts\bin\**\*.exe;-:f|artifacts\bin\**\msdia140.dll;-:f|artifacts\bin\**\pgort140.dll;-:f|artifacts\bin\*Tests\**;-f:|**\Microsoft.NET.Runtime.Emscripten**\tools\**.exe;-f:|**\CodeCoverage\**\*.*;
```

Comment:
@marcpopMSFT which syntax is the correct one? This line has both `-f:|` and `-:f|`

Reply:
I only see -:f in the previous version unless you meant a different line?

Reply:
Confirmed with the 1ES documentation that it's -:f